src/main/scala/chiseltest/legacy/backends/verilator/VerilatorCppHarnessGenerator.scala

```diff
@@ -190,8 +190,8 @@ int main(int argc, char **argv, char **env) {
 #endif
 #if VM_COVERAGE
     VL_PRINTF(\"Writing Coverage..\");
-    Verilated::mkdir("$targetDir/logs");
-    VerilatedCov::write("$targetDir/logs/coverage.dat");
+    Verilated::mkdir("$coverageDir");
+    VerilatedCov::write("$coverageDir/coverage.dat");
```

Comment:
How is this change related to this PR? Did you add this by accident?

Reply:
This was an intentional change. I realized when making the more global prototype that the coverage directory would always default to $targetDir/logs. I felt that it would be a more useful API to let to caller of the C++ harness generator fully specify the path to a coverage directory.

Reply:
I believe this should go into a separate PR

Reply:
Removed